CalculateTotal.js

```diff
@@ -0,0 +1,42 @@
+function getOrderTotal(items) {
+  var total = 0;
+  items.map((x) => {
+    total += x.cost * x.quantity;
+  });
+
+  // console.log("total", total);
+  return total;
+}
```

Comment:
The `reduce()` can be used here. The reduce method reduces the array to a single value. 

```suggestion
function getOrderTotal(items) {
  const total = items.reduce((result, item) => {
    return result + item.cost * item.quantity;
  }, 0);
  return total;
}
```